### PR TITLE
Address NPM audit alert 1762 (Uplift to stable)

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix https://github.com/brave/brave-browser/issues/14423

Uplift of https://github.com/brave/brave-core/pull/9267

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

**Review by petemill — APPROVED** [review]

**Comment by kjozwiak** [comment]
Looks like there's a few `plint` failures via `post-init` as per https://ci.brave.com/job/pr-brave-browser-1.26.x-prismjs-post-init/1/pylint. However, we never block uplifts on `plint` issues.

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.26.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
